src/config/configure.in

```diff
@@ -2140,11 +2140,11 @@ fi dnl hypre_using_um
 dnl CUDA
 if [test "$hypre_using_cuda" = "yes"]
 then
-   AC_CHECK_HEADERS(["${CUDA_HOME}/include/cuda.h"], [hypre_found_cuda=yes; HYPRE_CUDA_PATH=${CUDA_HOME}])
+   AC_CHECK_HEADER(["${CUDA_HOME}/include/cuda.h"], [hypre_found_cuda=yes; HYPRE_CUDA_PATH=${CUDA_HOME}])
```

Comment:
Do we make the change because we only check one header-file (as oppose to `AC_CHECK_HEADERS` checking multiple)? Thanks

Reply:
There was an error with 2.71 autoconf with the previous macro, but I don't remember exactly what the issue was.  I'm pretty sure the macro used in the fix is okay though.  I'll take another look at AC_CHECK_HEADERS.

Reply:
Here is the error message:
````
tux406 > config/update.sh 
HYPRE version number is the same as the current version
configure.in:2143: warning: AC_CHECK_HEADERS("${CUDA_HOME}/include/cuda.h"): you should use literals
./lib/autoconf/headers.m4:217: AC_CHECK_HEADERS is expanded from...
configure.in:2143: the top level
configure.in:2147: warning: AC_CHECK_HEADERS("${CUDA_PATH}/include/cuda.h"): you should use literals
./lib/autoconf/headers.m4:217: AC_CHECK_HEADERS is expanded from...
````